install/kubernetes/helm/istio/charts/pilot/templates/clusterrole.yaml

```diff
@@ -8,24 +8,14 @@ metadata:
     heritage: {{ .Release.Service }}
     release: {{ .Release.Name }}
 rules:
-- apiGroups: ["config.istio.io"]
+- apiGroups:
+  - config.istio.io
+  - rbac.istio.io
+  - security.istio.io
+  - networking.istio.io
+  - authentication.istio.io
   resources: ["*"]
-  verbs: ["*"]
-- apiGroups: ["rbac.istio.io"]
-  resources: ["*"]
-  verbs: ["get", "watch", "list"]
-- apiGroups: ["security.istio.io"]
-  resources: ["*"]
-  verbs: ["get", "watch", "list"]
-- apiGroups: ["networking.istio.io"]
-  resources: ["*"]
-  verbs: ["*"]
-- apiGroups: ["authentication.istio.io"]
-  resources: ["*"]
-  verbs: ["*"]
-- apiGroups: ["apiextensions.k8s.io"]
-  resources: ["customresourcedefinitions"]
```

Comment:
do we not need `get` for anything? I don't think we do but just checking

Reply:
Pilot doesn't. Istiod will eventually need this when Galley's processing pipeline is enabled.